src/current/v23.1/sso-sql.md

```diff
@@ -1,158 +1,257 @@
 
-For more details and examples, refer to [SSO to CockroachDB clusters using JWT](https://www.cockroachlabs.com/blog/sso-to-clusters-with-jwt/) in the CockroachDB blog.
+- SQL users/credentials:
 
-- **IdP:**
+	- Your SQL user must have the ability to update cluster settings. This permission is provided by either the [`admin` role]({% link {{ page.version.version }}/security-reference/authorization.md %}#admin-role) or the [`MODIFYCLUSTERSETTING` role option]({% link {{ page.version.version }}/security-reference/authorization.md %}#supported-privileges). This is required to designate an IdP as an external token issuer.
 
-	You must have the ability to create identities and issue access tokens formatted using JSON Web Token (JWT).
+	- A SQL user that corresponds with your external identity must be pre-provisioned on the cluster. To provision such users, you must have access to the [`admin` role]({% link {{ page.version.version }}/security-reference/authorization.md %}#admin-role).
 
-- **CockroachDB:**
+## Authenticate to your cluster
 
-	- **Self-Hosted**: You must have access to a cluster enabled with a valid [CockroachDB Enterprise license]({% link {{ page.version.version }}/enterprise-licensing.md %}).
+For access to a CockroachDB cluster, an identity in your external IdP must correspond to a SQL user on the target cluster.
 
-		See [Enterprise Trial –– Get Started]({% link {{ page.version.version }}/get-started-with-enterprise-trial.md %}) for help enabling your cluster with you enterprise license.
+The mapping from IdP identity to SQL username is defined in your cluster's [identity mapping](#identity-map-configuration).
 
-	- **CockroachDB {{ site.data.products.cloud }}**: You must have access to a [CockroachDB {{ site.data.products.dedicated }}cluster](https://www.cockroachlabs.com/docs/cockroachcloud/create-your-cluster).
+Refer to: [Create a SQL user]({% link {{ page.version.version }}/create-user.md %}).
 
-	- SQL users/credentials:
+{{site.data.alerts.callout_success}}
+This example uses [`cockroach sql`]({% link {{ page.version.version }}/cockroach-sql.md %}), but you can use any SQL client that supports sufficiently long passwords.
+{{site.data.alerts.end}}
 
-		- Your SQL user must have the ability to update cluster settings. This permission is provided by either the [`admin` role]({% link {{ page.version.version }}/security-reference/authorization.md %}#admin-role) or the [`MODIFYCLUSTERSETTING` role option]({% link {{ page.version.version }}/security-reference/authorization.md %}#supported-privileges). This is required to designate an IdP as an external token issuer.
 
-		- A SQL user that corresponds with your external identity must be pre-provisioned on the cluster. To provision such users, you must have access to the [`admin` role]({% link {{ page.version.version }}/security-reference/authorization.md %}#admin-role).
+1. Obtain a token.
 
-## Learn more
+Go to your cluster's DB Console and click the **generate token** button
```

Comment:
I'm not sure about throwing them directly at this UI. It will only work in some circumstances and they have to fully set up DB Console SSO as well as some bonus cluster settings for it to work.

Reply:
added some signposting